src/test/java/org/elasticsearch/search/aggregations/bucket/DateHistogramTests.java

```diff
@@ -100,16 +96,14 @@ public void setupSuiteScopeCluster() throws Exception {
         prepareCreate("empty_bucket_idx").addMapping("type", "value", "type=integer").execute().actionGet();
         List<IndexRequestBuilder> builders = new ArrayList<>();
         for (int i = 0; i < 2; i++) {
-            builders.add(client().prepareIndex("empty_bucket_idx", "type", ""+i).setSource(jsonBuilder()
-                    .startObject()
-                    .field("value", i*2)
-                    .endObject()));
+            builders.add(client().prepareIndex("empty_bucket_idx", "type", "" + i).setSource(
+                    jsonBuilder().startObject().field("value", i * 2).endObject()));
         }
-        builders.addAll(Arrays.asList(
-                indexDoc(1, 2, 1),  // date: Jan 2, dates: Jan 2, Feb 3
-                indexDoc(2, 2, 2),  // date: Feb 2, dates: Feb 2, Mar 3
+        builders.addAll(Arrays.asList(indexDoc(1, 2, 1), // date: Jan 2, dates:
+                                                         // Jan 2, Feb 3
```

Comment:
ups, badly behaved auto-formatting. Will change that back to original, but doesn't affect tests